webui/react/src/pages/FlatRuns/FlatRuns.tsx

```diff
@@ -363,7 +363,7 @@ const FlatRuns: React.FC<Props> = ({ projectId, workspaceId, searchId }) => {
             break;
           case V1LocationType.HYPERPARAMETERS:
           case V1LocationType.RUNHYPERPARAMETERS:
-            dataPath = `hyperparameters.${currentColumn.column.replace('hp.', '')}.val`;
+            dataPath = `hyperparameters.${currentColumn.column.replace('hp.', '')}`;
```

Comment:
The old way works for `HYPERPERAMETERS` so we should still keep it for the above case